apps/system/test/unit/accessibility_test.js

```diff
@@ -289,4 +313,67 @@ suite('system/Accessibility', function() {
         accessibility.quicknav.element.classList.contains('visible'));
     });
   });
+
+  suite('interaction hints', function() {
+    var screenNode, clock;
+    var realL10n = navigator.mozL10n;
+
+    setup(function() {
+      screenNode = document.createElement('div');
+      screenNode.id = 'screen';
+      document.body.appendChild(screenNode);
+
+      clock = this.sinon.useFakeTimers();
+
+      navigator.mozL10n = MockL10n; // jshint ignore:line
+    });
+
+    teardown(function() {
+      screenNode.parentNode.removeChild(screenNode);
+      clock.restore();
+      navigator.mozL10n = realL10n;
+    });
+
+    test('handle vc change event with hints enabled', function() {
+      var stubSetHintsTimeout = this.sinon.stub(accessibility,
+        'setHintsTimeout');
+      // This is where MozL10n was necessary in place of the stubbed
+      // accessibility.speak method
+      accessibility.handleAccessFuOutput(vcChangeHintsEnabledDetails);
+      clock.tick(2000);
+      assert.isTrue(stubSetHintsTimeout.called);
+      assert.isTrue(stubSetHintsTimeout.calledWith(
+        vcChangeHintsEnabledDetails.data.concat(
+          vcChangeHintsEnabledDetails.options.hints)));
+    });
+
+    test('make sure hints are spoken', function() {
+      var stubSpeak = this.sinon.stub(accessibility, 'speak');
+      accessibility.setHintsTimeout(aDataPlusHints);
+      clock.tick(accessibility.HINTS_TIMEOUT+50);
```

Comment:
I know it's a little more work, but could we wrap this test in it's own suite with clock setup and teardown to not affect the ones that will not need it?